src/tests/ftest/soak/ci_integration.py

```diff
@@ -121,7 +121,7 @@ def test_java_hadoop_it(self):
 
 
         # run intergration-test
-        openjdk = "/usr/lib/jvm/java-{}-openjdk-{}-0.el7_8.x86_64/jre/lib/amd64/libjsig.so".format(version[:-8], version)
+        openjdk = "/usr/lib/jvm/java-{}-openjdk-{}-0.el7_9.x86_64/jre/lib/amd64/libjsig.so".format(version[:-8], version)
```

Comment:
While I can appreciate that this might just be a stepping stone, to get the basics of the test working, this demonstrates how fragile hard-coding version bits is going to be.  Ultimately this is going to need to be determined dynamically/automatically.

Reply:
If you have Java installed via package manager, like yum or apt, you may try https://github.com/daos-stack/daos/blob/master/src/client/java/daos-java/find_java_home.sh to find Java home. Then use, <Java home>/jre/lib/adm64/libjsig.so.

Reply:
I agree. I was just taking my chances while I was working on other things. But yeah it seems like this need to be completely dynamic.